./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr1/chr1.sort.fix.brecal.bam > ./TP00005/chr1/chr1.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr1/chr1.pileup ./TP00005/chr1/chr1.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr1/chr1.genotyper.vcf -pileup ./TP00005/chr1/chr1.snp.indel -o  ./TP00005/chr1/chr1.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr1/chr1.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr1 -o ./TP00005/chr1/chr1.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr2/chr2.sort.fix.brecal.bam > ./TP00005/chr2/chr2.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr2/chr2.pileup ./TP00005/chr2/chr2.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr2/chr2.genotyper.vcf -pileup ./TP00005/chr2/chr2.snp.indel -o  ./TP00005/chr2/chr2.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr2/chr2.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr2 -o ./TP00005/chr2/chr2.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr3/chr3.sort.fix.brecal.bam > ./TP00005/chr3/chr3.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr3/chr3.pileup ./TP00005/chr3/chr3.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr3/chr3.genotyper.vcf -pileup ./TP00005/chr3/chr3.snp.indel -o  ./TP00005/chr3/chr3.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr3/chr3.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr3 -o ./TP00005/chr3/chr3.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr4/chr4.sort.fix.brecal.bam > ./TP00005/chr4/chr4.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr4/chr4.pileup ./TP00005/chr4/chr4.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr4/chr4.genotyper.vcf -pileup ./TP00005/chr4/chr4.snp.indel -o  ./TP00005/chr4/chr4.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr4/chr4.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr4 -o ./TP00005/chr4/chr4.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr5/chr5.sort.fix.brecal.bam > ./TP00005/chr5/chr5.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr5/chr5.pileup ./TP00005/chr5/chr5.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr5/chr5.genotyper.vcf -pileup ./TP00005/chr5/chr5.snp.indel -o  ./TP00005/chr5/chr5.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr5/chr5.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr5 -o ./TP00005/chr5/chr5.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr6/chr6.sort.fix.brecal.bam > ./TP00005/chr6/chr6.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr6/chr6.pileup ./TP00005/chr6/chr6.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr6/chr6.genotyper.vcf -pileup ./TP00005/chr6/chr6.snp.indel -o  ./TP00005/chr6/chr6.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr6/chr6.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr6 -o ./TP00005/chr6/chr6.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr7/chr7.sort.fix.brecal.bam > ./TP00005/chr7/chr7.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr7/chr7.pileup ./TP00005/chr7/chr7.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr7/chr7.genotyper.vcf -pileup ./TP00005/chr7/chr7.snp.indel -o  ./TP00005/chr7/chr7.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr7/chr7.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr7 -o ./TP00005/chr7/chr7.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr8/chr8.sort.fix.brecal.bam > ./TP00005/chr8/chr8.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr8/chr8.pileup ./TP00005/chr8/chr8.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr8/chr8.genotyper.vcf -pileup ./TP00005/chr8/chr8.snp.indel -o  ./TP00005/chr8/chr8.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr8/chr8.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr8 -o ./TP00005/chr8/chr8.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr9/chr9.sort.fix.brecal.bam > ./TP00005/chr9/chr9.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr9/chr9.pileup ./TP00005/chr9/chr9.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr9/chr9.genotyper.vcf -pileup ./TP00005/chr9/chr9.snp.indel -o  ./TP00005/chr9/chr9.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr9/chr9.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr9 -o ./TP00005/chr9/chr9.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr10/chr10.sort.fix.brecal.bam > ./TP00005/chr10/chr10.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr10/chr10.pileup ./TP00005/chr10/chr10.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr10/chr10.genotyper.vcf -pileup ./TP00005/chr10/chr10.snp.indel -o  ./TP00005/chr10/chr10.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr10/chr10.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr10 -o ./TP00005/chr10/chr10.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr11/chr11.sort.fix.brecal.bam > ./TP00005/chr11/chr11.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr11/chr11.pileup ./TP00005/chr11/chr11.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr11/chr11.genotyper.vcf -pileup ./TP00005/chr11/chr11.snp.indel -o  ./TP00005/chr11/chr11.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr11/chr11.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr11 -o ./TP00005/chr11/chr11.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr12/chr12.sort.fix.brecal.bam > ./TP00005/chr12/chr12.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr12/chr12.pileup ./TP00005/chr12/chr12.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr12/chr12.genotyper.vcf -pileup ./TP00005/chr12/chr12.snp.indel -o  ./TP00005/chr12/chr12.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr12/chr12.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr12 -o ./TP00005/chr12/chr12.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr13/chr13.sort.fix.brecal.bam > ./TP00005/chr13/chr13.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr13/chr13.pileup ./TP00005/chr13/chr13.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr13/chr13.genotyper.vcf -pileup ./TP00005/chr13/chr13.snp.indel -o  ./TP00005/chr13/chr13.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr13/chr13.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr13 -o ./TP00005/chr13/chr13.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr14/chr14.sort.fix.brecal.bam > ./TP00005/chr14/chr14.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr14/chr14.pileup ./TP00005/chr14/chr14.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr14/chr14.genotyper.vcf -pileup ./TP00005/chr14/chr14.snp.indel -o  ./TP00005/chr14/chr14.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr14/chr14.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr14 -o ./TP00005/chr14/chr14.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr15/chr15.sort.fix.brecal.bam > ./TP00005/chr15/chr15.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr15/chr15.pileup ./TP00005/chr15/chr15.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr15/chr15.genotyper.vcf -pileup ./TP00005/chr15/chr15.snp.indel -o  ./TP00005/chr15/chr15.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr15/chr15.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr15 -o ./TP00005/chr15/chr15.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr16/chr16.sort.fix.brecal.bam > ./TP00005/chr16/chr16.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr16/chr16.pileup ./TP00005/chr16/chr16.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr16/chr16.genotyper.vcf -pileup ./TP00005/chr16/chr16.snp.indel -o  ./TP00005/chr16/chr16.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr16/chr16.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr16 -o ./TP00005/chr16/chr16.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr17/chr17.sort.fix.brecal.bam > ./TP00005/chr17/chr17.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr17/chr17.pileup ./TP00005/chr17/chr17.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr17/chr17.genotyper.vcf -pileup ./TP00005/chr17/chr17.snp.indel -o  ./TP00005/chr17/chr17.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr17/chr17.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr17 -o ./TP00005/chr17/chr17.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr18/chr18.sort.fix.brecal.bam > ./TP00005/chr18/chr18.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr18/chr18.pileup ./TP00005/chr18/chr18.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr18/chr18.genotyper.vcf -pileup ./TP00005/chr18/chr18.snp.indel -o  ./TP00005/chr18/chr18.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr18/chr18.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr18 -o ./TP00005/chr18/chr18.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr19/chr19.sort.fix.brecal.bam > ./TP00005/chr19/chr19.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr19/chr19.pileup ./TP00005/chr19/chr19.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr19/chr19.genotyper.vcf -pileup ./TP00005/chr19/chr19.snp.indel -o  ./TP00005/chr19/chr19.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr19/chr19.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr19 -o ./TP00005/chr19/chr19.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr20/chr20.sort.fix.brecal.bam > ./TP00005/chr20/chr20.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr20/chr20.pileup ./TP00005/chr20/chr20.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr20/chr20.genotyper.vcf -pileup ./TP00005/chr20/chr20.snp.indel -o  ./TP00005/chr20/chr20.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr20/chr20.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr20 -o ./TP00005/chr20/chr20.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr21/chr21.sort.fix.brecal.bam > ./TP00005/chr21/chr21.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr21/chr21.pileup ./TP00005/chr21/chr21.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr21/chr21.genotyper.vcf -pileup ./TP00005/chr21/chr21.snp.indel -o  ./TP00005/chr21/chr21.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr21/chr21.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr21 -o ./TP00005/chr21/chr21.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chr22/chr22.sort.fix.brecal.bam > ./TP00005/chr22/chr22.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chr22/chr22.pileup ./TP00005/chr22/chr22.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chr22/chr22.genotyper.vcf -pileup ./TP00005/chr22/chr22.snp.indel -o  ./TP00005/chr22/chr22.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chr22/chr22.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chr22 -o ./TP00005/chr22/chr22.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chrX/chrX.sort.fix.brecal.bam > ./TP00005/chrX/chrX.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chrX/chrX.pileup ./TP00005/chrX/chrX.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chrX/chrX.genotyper.vcf -pileup ./TP00005/chrX/chrX.snp.indel -o  ./TP00005/chrX/chrX.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chrX/chrX.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chrX -o ./TP00005/chrX/chrX.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chrY/chrY.sort.fix.brecal.bam > ./TP00005/chrY/chrY.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chrY/chrY.pileup ./TP00005/chrY/chrY.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chrY/chrY.genotyper.vcf -pileup ./TP00005/chrY/chrY.snp.indel -o  ./TP00005/chrY/chrY.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chrY/chrY.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chrY -o ./TP00005/chrY/chrY.final.indel 
./bin/samtools mpileup -f  /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa ./TP00005/chrM/chrM.sort.fix.brecal.bam > ./TP00005/chrM/chrM.pileup && perl  ./bin/pileup_analyse.pl ./TP00005/chrM/chrM.pileup ./TP00005/chrM/chrM.snp.indel && perl ./bin/fitter_indel_bias.pl -vcf ./TP00005/chrM/chrM.genotyper.vcf -pileup ./TP00005/chrM/chrM.snp.indel -o  ./TP00005/chrM/chrM.indel1 && perl ./bin/fitter_indel_homo2.pl -vcf ./TP00005/chrM/chrM.indel1 -g /ifs1/ST_RNA/USER/zhupengyuan/ref/Homo_genome/genome.fa -chr chrM -o ./TP00005/chrM/chrM.final.indel 
